dotcom-rendering/src/server/prod-server.ts

```diff
@@ -21,13 +21,20 @@ import { recordBaselineCloudWatchMetrics } from './lib/aws/metrics-baseline';
 import { getContentFromURLMiddleware } from './lib/get-content-from-url';
 import { logger } from './lib/logging';
 
+const getPageIdFromBody = (body: unknown): string => {
+	const baseBody = body as BaseRequestBody;
+	return baseBody.pageId || 'no-page-id-found';
+};
+
 // Middleware to track route performance using 'response-time' lib
 // Usage: app.post('/Article', logRenderTime, renderArticle);
 const logRenderTime = responseTime(
-	({ body }: Request, _: Response, time: number) => {
-		const { pageId = 'no-page-id-found' } = body as FEArticleType;
+	({ body, path }: Request, _: Response, time: number) => {
 		logger.info('Page render time', {
-			pageId,
+			req: {
+				path,
```

Comment:
Looking at our logging elsewhere, we seem to use `req.url` - should we replicate that here?